contractcourt/briefcase.go

```diff
@@ -533,7 +535,9 @@ func (b *boltArbitratorLog) FetchUnresolvedContracts() ([]ContractResolver, erro
 // swapped out, or resolved.
 //
 // NOTE: Part of the ContractResolver interface.
-func (b *boltArbitratorLog) InsertUnresolvedContracts(resolvers ...ContractResolver) error {
+func (b *boltArbitratorLog) InsertUnresolvedContracts(closure func(
```

Comment:
If the closure always persists the resolver report, can't the report just be passed in here? To make it clearer what is happening?